uiowa/uiowa
# Create cuhal lab site and adjust the UMC table output.

I noticed some odd output when creating this site. The columns were off so I changed how the rows were generated using array keys. I swear this didn't work before in BLT 11. Also fixes #2975 and updated the related domains header to singular because we only check for one.

### Testing
- `blt umc foo.uiowa.edu --no-commit --simulate`
- See that uiowa.edu on uiowa01 is not marked as related.
- `blt umc foo.hr.uiowa.edu --no-commit --simulate`
- See that hr.uiowa.edu is marked as related on uiowa.
- `blt umc foo.lab.uiowa.edu --no-commit --simulate`
- Table output should be something like:
```
+-------------+-----+------+-----------------+----------------------+
| Application | DBs | SANs | SSL - Coverage  | SSL - Related Domain |
+-------------+-----+------+-----------------+----------------------+
| uiowa       | 158 | 67   | *.lab.uiowa.edu |                      |
| uiowa01     | 118 | 65   | *.lab.uiowa.edu |                      |
| uiowa02     | 114 | 36   | *.lab.uiowa.edu |                      |
+-------------+-----+------+-----------------+----------------------+
```
And not something like: 
```
+-------------+-----+------+-------------------+-----------------------+-------------------+--+
| Application | DBs | SANs | SSL - Coverage    | SSL - Related Domains |                   |  |
+-------------+-----+------+-------------------+-----------------------+-------------------+--+
| uiowa       | 158 | 67   | *.sites.uiowa.edu |                       |                   |  |
| uiowa01     | 118 | 63   | *.sites.uiowa.edu | 65                    | *.sites.uiowa.edu |  |
| uiowa02     | 113 | 36   | *.sites.uiowa.edu |                       |                   |  |
+-------------+-----+------+-------------------+-----------------------+-------------------+--+
```
